dist/js/select2.full.js

```diff
@@ -3444,7 +3444,7 @@ S2.define('select2/data/ajax',[
       }, function () {
         // Attempt to detect if a request was aborted
         // Only works if the transport exposes a status property
-        if ($request.status && $request.status === '0') {
+        if ('status' in $request && $request.status === 0) {
```

Comment:
Is this not going to break support for all browsers that have '0' ? shouldnt this be `$request.status === 0 || $request.status === '0'`?

Reply:
For those who are using timeout : if ('status' in $request && $request.status === 0 && $request.statusText != "timeout")